libtextsecure/message_receiver.js

```diff
@@ -765,6 +765,9 @@ MessageReceiver.prototype.extend({
         } else if (decrypted.flags & textsecure.protobuf.DataMessage.Flags.EXPIRATION_TIMER_UPDATE ) {
             decrypted.body = null;
             decrypted.attachments = [];
+        } else if (decrypted.flags & textsecure.protobuf.DataMessage.Flags.PROFILE_KEY_UPDATE) {
```

Comment:
I'm surprised that our flags handling uses `else if` given that our flags bitpacking technique can result in multiple flags. Are we really sure that we'll only get one of these at a time?

Reply:
In practice this has always been the case, yes.